Database opzetten

Qwarder opened the pull request:
Database volgens DB schema

Qwarder committed:
Database opzetten

jasperweyne commented on src/Entity/LokaalPersoon.php:
```diff
@@ -0,0 +1,172 @@
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $Mail_adres;
```
Please refactor to use english language, as this makes the codebase more consistent. Please use [PSR-12](https://www.php-fig.org/psr/psr-12/) naming as well

jasperweyne commented on src/Entity/LokaalPersoon.php:
```diff
@@ -0,0 +1,172 @@
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
```
Please use a UUID instead of a int, as an int may be guessed and could pose a security issue, where UUID is random

jasperweyne commented on src/Entity/LokaalPersoon.php:
```diff
@@ -0,0 +1,172 @@
+/**
+ * @ORM\Entity(repositoryClass=LokaalPersoonRepository::class)
+ */
+class LokaalPersoon
```
Since external users can't or shouldn't be included in the database, and internal users are stored in the OIDC store, this class should be removed and later implemented through fetching over an API

jasperweyne reviewed (changes requested):
Looking good for first PR version, I'll review further after the changes

Qwarder committed:
Change request update

Naming from Dutch to English.
Dropped LokaalPersoon to be replaced with account system
updated ID to UUID

Qwarder committed:
Fixed migrations

Updated symfony
Downgraded dbal

mkrugr reviewed (changes requested):
TransactionSet -> TransactionGroup.
GeefGeldController -> DeclaractionController.

Furthermore fine code.

Qwarder committed:
Name changes

last few changes to english

Qwarder committed:
Clean

Hopefully last dutch cleaning

Qwarder committed:
Migration duplication

merged migrations

mkrugr reviewed (approved):
Perfection.
